File: Modul-3/PRAK304/PRAK304-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>

int main(){
    int a;
    scanf("%d",&a);
    if(a>0&&a<10){
        printf("satuan");
    }
    else if(a==0){
        printf("nol");
    }
    else if(a>=100){
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
    else if(a>=20){
        printf("puluhan");
    }
    else if(a>=10){
        printf("belasan");
    }
    return 0;
}

File: Modul-3/PRAK304/PRAK304-2310817210010-Harry-Pratama-Yunus.py
a=int(input())
if a>0 and a<10:
    print("Satuan")
elif a==0:
    print("Nol")
elif a>=100:
    print("Anda Menginput Melebihi Limit Bilangan")
elif a>=20:
    print("Puluhan")
elif a>=10:
    print("Belasan")
